sky/serve/service_spec.py

```diff
@@ -176,7 +211,6 @@ def __repr__(self) -> str:
             Readiness probe method:           {self.probe_str()}
             Readiness initial delay seconds:  {self.initial_delay_seconds}
             Replica autoscaling policy:       {self.policy_str()}
```

Comment:
```suggestion
            Replica autoscaling policy:       {self.policy_str()}\
```
nit: Add `\` to remove the `\n` at the end of multi-line str